src/Domain/Donation.php

```diff
@@ -1719,4 +1745,20 @@ public function thisIsInDateRangeToConfirm(DateTimeImmutable $now): bool
             $preAuthorizationDate <= $now &&
             $preAuthorizationDate->add(new \DateInterval('P1M')) >= $now;
     }
+
+    public function getRefundedAt(): ?DateTimeImmutable
+    {
+        return $this->refundedAt;
+    }
+
+    public function getTipRefundAmount(): ?Money
+    {
+        if ($this->tipRefundAmount === null) {
+            return null;
+        }
+
+        // cheating a little by asserting that currency is GBP, since it is always likely to be.
```

Comment:
```suggestion
        // @todo-multi-currency cheating a little by asserting that currency is GBP, since it is always likely to be.
```
I think this is fine, but shall we start a new todo marker to help whenever we have a use case for international support again?

Reply:
Happy to commit that change, although I was also wondering in the last months about if we should go the other way - if multi currency is unlikely enough to happen that we should do more to simplify the code by removing the support for multi currency that's there, or at least making a policy that we won't support it in new code.

So for instance we'd had that policy before we created the Money VO it wouldn't have had a currency property, it would just have been documented to always refer to GBP.

Reply:
My expectation is that it will happen, just probably not this year.

Reply:
OK, will try and keep building mostly things based on the idea that we support a list of currencies that happens to currently include only GBP without relying on that being the only one.